lib/items.rb

```diff
@@ -0,0 +1,20 @@
+class Item
+
+  attr_reader :id,
+              :name,
+              :description,
+              :unit_price,
+              :merchant_id,
+              :created_at,
+              :updated_at
+
+  def initialize (data)
+    @id          = data[:id]
+    @name        = data[:name]
+    @description = data[:description]
+    @unit_price  = data[:unit_price]
+    @merchant_id = data[:merchant_id]
+    @created_at  = data[:created_at]
+    @updated_at  = data[:updated_at]
+  end
+end
```

Comment:
Looks good Khoa - Isn't there also a `unit_price_to_dollars` method in this class?

Reply:
yeah, but i think that will go in the items repo file

Reply:
You sure it's not in the Item class? Looking over the iteration 0 guidance it reads like it is. I guess we can always move things around later.

Reply:
> You sure it's not in the Item class? Looking over the iteration 0 guidance it reads like it is. I guess we can always move things around later.

I think you are correct in that the `unit_price_to_dollars` lies in the `item.rb` file. I can hop on a call if need be! 
<img width="736" alt="image" src="https://user-images.githubusercontent.com/63377504/103590540-548f8c80-4eab-11eb-8043-0a7a76d0140c.png">

Reply:
I see where Tommy is coming from... there's a method following the list of Item instances with an extra method called unit_price_to_dollars